packages/dma-contracts/contracts/core/OperationExecutor.sol

```diff
@@ -84,12 +84,16 @@ contract OperationExecutor is IERC3156FlashBorrower, IFlashLoanRecipient, UseSto
    *
    * @param calls List of action calls to be executed.
    */
-  function executeOp(Call[] memory calls) public payable returns (bytes32) {
+  function executeOp(Call[] memory calls) external payable returns (bytes32) {
     aggregate(calls);
 
     bytes32[] memory actions = storeInSlot("actions").returnStoredArray();
     bytes32 operationName = getOperation(keccak256(abi.encodePacked(actions)));
 
+    /* Transient storage cleared in case of `executeOp ` being called multiple times in one transaction */
+    storeInSlot("actions").clearStoredArray();
```

Comment:
I'd extract rather have specialized libraries for actions and transaction in which those strings are embedded in the library, so we could use `ActionsStorage.clear()` and `TransactionStorage.clear()`. If that is too much of a change at this point, I'd suggest at least extracting the strings as constants